static/assets/js/create_heritage_item.js

```diff
@@ -1,96 +1,9 @@
 $(function() {
-
-    var geo_location_pin_map = new Maplace({
-        map_div: '#create_new_heritage_item_pin_gmap',
-        type: 'marker',
-        locations: [{
-            lat: 38.9637,
-            lon: 35.2433,
-            draggable: true,
-            zoom: 5
-        }]
-    }).Load();
-
-    var geo_location_circle_map = new Maplace({
-        map_div: '#create_new_heritage_item_circle_gmap',
-        type: 'circle',
-        locations: [{
-            lat: 38.9637,
-            lon: 35.2433,
-            circle_options: {
-                radius: 150000,
-                editable: true
-            },
-            zoom: 5,
-            draggable: true
-        }]
-    }).Load();
-
-    geo_location_polygon_map = new Maplace({
-        map_div: '#create_new_heritage_item_polygon_gmap',
-        type: 'polygon',
-        locations: [{
-            lat: 39.5,
-            lon: 32.2433,
-            draggable: true,
-            zoom: 5
-        },
-        {
-            lat: 39.5,
-            lon: 38.2433,
-            draggable: true,
-            zoom: 5
-        },
-        {
-            lat: 38,
-            lon: 35.2433,
-            draggable: true,
-            zoom: 5
-        }]
-    }).Load();
-
-    var geo_location_polyline_map = new Maplace({
-        map_div: '#create_new_heritage_item_path_polyline_gmap',
-        type: 'polyline',
-        locations: [{
-            lat: 37,
-            lon: 31,
-            draggable: true,
-            zoom: 5
-        },
-        {
-            lat: 39.5,
-            lon: 33,
-            draggable: true,
-            zoom: 5
-        },
-        {
-            lat: 41,
-            lon: 34,
-            draggable: true,
-            zoom: 5
-        },
-        {
-            lat: 37,
-            lon: 36,
-            draggable: true,
-            zoom: 5
-        },
-        {
-            lat: 40,
-            lon: 38,
-            draggable: true,
-            zoom: 5
-        },
-        {
-            lat: 38.9637,
-            lon: 40,
-            draggable: true,
-            zoom: 5
-        }]
-    }).Load();
-
-
+    var geo_location_polyline_map;
+    var geo_location_polygon_map;
+    var geo_location_circle_map;
+    geo_location_pin_map = 2;
+    map = 1;
```

Comment:
:) yes, I am aware of these strange assignments and changed them already in my local env.